tests/scripts/test_translate_ciphers_names.py

```diff
@@ -172,15 +183,20 @@ def test_all_common():
         "PSK-AES256-CBC-SHA",
     ]
 
-    for i in range(len(m_ciphers)):
+    for i, m_cipher in enumerate(m_ciphers):
```

Comment:
Since we're talking about Python style, I think this is one case where [zip](https://docs.python.org/3.8/library/functions.html#zip) would be more pythonic:
````
for m, g_exp, o_exp in zip(m_ciphers, g_ciphers, o_ciphers):
    g = translate_gnutls(m)
    assert_equal(g, g_exp)
    # etc
```